packages/backend/src/open_payments/quote/routes.ts

```diff
@@ -80,7 +80,7 @@ async function createQuote(
     }
 
     ctx.status = 201
-    ctx.body = quoteToBody(quoteOrErr)
+    ctx.body = await quoteToBody(quoteOrErr)
```

Comment:
```suggestion
    ctx.body = await quoteOrErr.toOpenPaymentsType()
```
Remove `quoteToBody`